src/HierarchyAnalysisAlgorithm.cc

```diff
@@ -262,7 +262,7 @@ void HierarchyAnalysisAlgorithm::EventAnalysisOutput(const LArHierarchyHelper::M
 
                 // Make sure the cluster has some hits
                 const int n3DHits(pCluster3D->GetNCaloHits());
-                if (n3DHits == 0)
+                if (n3DHits <= 1)
```

Comment:
Could you remind me what was the motivation for this change? I seem to recall this was to fix a bug but I wasn't able to find it in slack! Thanks!